bobcat/bb-cumber/src/main/java/com/cognifide/qa/bb/cumber/Bobcumber.java

```diff
@@ -58,10 +61,14 @@
 
   private static final String COLON = ":";
 
-  public static final String FEATURE = "feature";
+  private static final String FEATURE = "feature";
+
+  private static final String RERUN_FAILED_TESTS_CLASS_NAME = "com.cognifide.qa.bobcumber.RerunFailedTests";
```

Comment:
instad of providing such string here please get fully qualified class name as string from this class. This will prevent mistakes during package changes etc